### DISCUSSION TITLE
what compiler and language version is used for C++ code?

### DISCUSSION THREAD

**Discussion by sehe** [geeksblabla/blanat#184] [Q&A] [answered]
E.g. can c++23 features be used?

**Comment by sehe** [comment]
Apparently GCC 12.2.0 https://github.com/geeksblabla/blanat/discussions/35

**Comment by yousfiSaad** [comment, marked as answer]
What about the optimization flags, do you use -O3 flag ?

Edit:
Yes the flag -O3 is used, for more informations see https://github.com/geeksblabla/blarun/blob/5d81dacacd26056523c5a69b757d9059f85223d0/src/main.rs#L265

**Reply by sehe** [reply]
Would it be possible to update to -Ofast -std=c++20 or would it invalidating existing entries? Anyways, those would jave been sensible options here :)

EDIT: Oh I see you were not maintainer. Excellent find, thanks for your in formations

**Reply by yousfiSaad** [reply]
You're welcome
